temporalcli/commands.env_test.go

```diff
@@ -7,20 +7,14 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
-// TODO(cretz): To test:
-// * Env var actually sets CLI arg
```

Comment:
Env var tests were previously added with the workflow tests (i.e. `TestWorkflow_Execute_EnvVars`)